src/gui/properties/propertieswidget.ui

```diff
@@ -174,7 +174,7 @@
                </widget>
               </item>
               <item row="2" column="2">
-               <widget class="QLabel" name="label_upload_speed">
+               <widget class="QLabel" name="labelUpSpeed">
```

Comment:
Let's stick to a single style. I mean to use either type infix or type suffix in widget names.